src/components/ScholarshipDetailCard.jsx

```diff
@@ -127,25 +127,30 @@ export default function ScholarshipDetailCard({ scholarship }) {
   return (
     <Card className={classes.detailSection}>
       <Typography variant="h4">{name}</Typography>
-      <Typography variant="h5">{organization}</Typography>
-      <Box className={classes.actionSection}>
-        <Button
-          component={Link}
-          href={website}
-          variant="contained"
-          color="primary"
-          className={classes.applyBtn}
-          startIcon={<Send />}>
-          Apply
-        </Button>
-
-        <Button
-          variant="outlined"
-          className={classes.shareBtn}
-          startIcon={<Share />}
-          onClick={shareFn}>
-          Share
-        </Button>
+      <Typography variant="h5" className={review ? classes.description : ''}>
+        {organization}
+      </Typography>
+      <Box className={!review ? classes.actionSection : ''}>
+        {!review && (
+          <>
+            <Button
+              component={Link}
+              href={website}
+              variant="contained"
+              color="primary"
+              className={classes.applyBtn}
+              startIcon={<Send />}>
+              Apply
+            </Button>
+            <Button
+              variant="outlined"
+              className={classes.shareBtn}
+              startIcon={<Share />}
+              onClick={shareFn}>
+              Share
+            </Button>{' '}
+          </>
+        )}
       </Box>
 
       <Typography
```

Comment:
A bit tangential but instead of setting `marginBottom` in the styling I think setting the `paragraph` property here should give adequate spacing beneath and we can get rid of the description CSS style.